packages/lesmis/src/components/DashboardExportModal/DashboardExportModal.js

```diff
@@ -34,6 +34,10 @@ const MuiButton = styled(MuiIconButton)`
   color: #666666;
 `;
 
+const EmptyView = styled.div`
+  height: 50vh;
```

Comment:
Where does 50vh come from? Seems pretty magic to me, and I can't see what it relates to...

Reply:
Yeah it is a very special case for `LoadingContainer`. `EmptyView` will help us to control the height of `LoadingContainer` dynamically.
The first thing is the height of `LoadingContainer` is the height of its child component. The existing problem is `LoadingContainer` current child is`ExportView`, and its height is fixed, so when loading, if client has a small screen they cannot see the loading circular progress.
I choose to adapt the behaviour of `LoadingContainer` since it got used for many place.

Reply:
Right I get it!

How about something like this? https://github.com/beyondessential/tupaia/pull/4048/files

The `EmptyView` wasn't as clear to me as an "isLoading" prop that defines the different styles

Reply:
That's much clearer @edmofro thanks! Nice one.